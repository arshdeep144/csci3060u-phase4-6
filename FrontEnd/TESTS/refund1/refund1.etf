10 admin01 AD 99999
05 user01 user02 65
00 admin01 AD 99999
