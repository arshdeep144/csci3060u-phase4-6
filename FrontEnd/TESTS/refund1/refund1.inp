login
admin01
refundcredit
user01
user02
65
logout