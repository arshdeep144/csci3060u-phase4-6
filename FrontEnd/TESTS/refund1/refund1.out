Welcome to the Online Auction System! Please login to Proceed
Please enter your username: 
Welcome to the main menu. Please enter a command to continue: 
Please enter username to send refund to: Please enter seller username: Please enter how much credit you would like to refund
Adding credits...
Subtracting credits...
New credits for Seller are: 956 and new credits for user are: 1094
Welcome to the main menu. Please enter a command to continue: 
